packages/magento-cart-items/Api/CartItem.graphql

```diff
@@ -24,6 +24,9 @@ fragment CartItem on CartItemInterface @injectable {
     price {
       ...Money
     }
+    price_including_tax {
+      ...Money
+    }
```

Comment:
Can we use the `@include` directive based on the `cartDisplayPricesInclTax` config for this?